docs/how-to/enable-intel-gpu.rst

```diff
@@ -0,0 +1,97 @@
+.. _enable_intel_gpu:
+
+Enable Intel GPUs
+=============================
+
+This guide describes how to configure DSS to utilise the Intel GPUs on your machine. This can be done by enabling the Intel device plugin on your MicroK8s cluster.
+
+Prerequisites
+-------------
+
+* :ref:`MicroK8s is installed <set_microk8s>`.
+* :ref:`DSS CLI is installed <install_DSS_CLI>`.
+* The Intel GPU is recognised by your machine. This can be confirmed by running `intel_gpu_top` from the `intel-gpu-tools` package. For discrete GPUs on older versions of Ubuntu like 22.04, you may need to perform additional steps like install a HWE kernel.
+
+Install the kubectl snap
+---------------------------
+First, install the `kubectl` snap package:
```

Comment:
Installing kubectl should not be part of this guide.

Reply:
I would drop this entire section

Reply:
Note that the `kubectl` snap is a requirement in our case, see [this comment](https://github.com/canonical/data-science-stack/issues/148#issuecomment-2273234977). What do you think @kimwnasptd?

Reply:
unfortunately yes, because of a bug in `microk8s.kubectl` we will have to install this binary

Reply:
I get that, i.e., the user needs to use `kubectl` but explaining how to install it is not part of the guide. Then, add `kubectl` as part of the requirements/prerequisites with a link for installation/getting started. 

Caveat: in how-tos, we need to be very use-case specific and explain the things that really refer to the goal of the guide. We also expect some prior knowledge from the user. E.g., if you (user) don't know what is kubectl and how to install it, go somewhere else to find that info and then come back.

Reply:
That makes sense, I'll add it as a requirement and mention that we want to install the snap package specifically.

Reply:
The problem then is where should we put these parts that are necessary for kubectl to communicate with MicroK8s:
```
  mkdir -p ~/.kube
  microk8s config > ~/.kube/config
```

Reply:
The section is dropped. We do not add any of this info. It does not seem to be specific to the Intel case, right? It is just part of the kubectl installation phase. Or am I wrong?

Reply:
It's specifically for when you use `kubectl` to communicate with `microk8s`. I don't there is a specific part of documentation that shows this, because in most cases we just use `microk8s.kubectl`.

Reply:
I see, but that's not related to DSS at all then IMO. If that is not documented anywhere, then it is a problem (that ideally we should report) of `kubectl` / `microK8s` teams and they should address.

If you think that avoiding presenting this info is going to block users to enable their Intel GPUs, then let's see how we add it here, but we should avoid this sort of documentation practices. It feels to me like to do a certain recipe that requires a pan, you find the instructions on how to build a pan in that recipe. That breaks the independence across products (which is already happening a lot within Canonical stack, but we need to start avoiding it).

@kimwnasptd  thoughts?

Reply:
Agreed.

1. The kubectl tool would've been installed a priori when deploying microk8s 
2. All kubectl configurations are not related at all to intel gpus, as Angel says.
3. There should be a documentation on kubectl on how to do this, if it's not, then there is an issue with kubectl's docs, as Angel says.